templates/participants.html

```diff
@@ -72,7 +72,11 @@ <h1>Wszyscy uczestnicy</h1>
                   {{ person.has_letter | qualified_mark }}
                 </td>
                 <td>
-                  {{ person.status }}
+                  {% if person.status == 'Z' %}
```

Comment:
I told you to add the colors to the workshop status too so that their behaviour is in sync...